hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/DeadNodeDetector.java

```diff
@@ -475,6 +475,7 @@ public synchronized void addNodeToDetect(DFSInputStream dfsInputStream,
       datanodeInfos.add(datanodeInfo);
     }
 
+    LOG.warn("Add datanode {} to suspectAndDeadNodes", datanodeInfo);
```

Comment:
One case: when a lot of invalid relicas, will the log flood?

Reply:
use DEBUG as default now